### ISSUE TITLE
This task uses Node 6 execution handler, which will be deprecated soon

### ISSUE THREAD

**Issue by eirikwah** [Dinomite-Studios/unity-azure-pipelines-tasks#190]
When using the "Unity Get Project Version" task, we get a warning:

> This task uses Node 6 execution handler, which will be deprecated soon. If you are the developer of the task - please consider the migration guideline to Node 10 handler - https://aka.ms/migrateTaskNode10 . If you are the user - feel free to reach out to the owners of this task to proceed on migration.

Full log: 

```
Starting: Unity Get Project Version
==============================================================================
Task         : Unity Get Project Version
Description  : Gets the project version of a Unity project.
Version      : 1.0.32
Author       : Dinomite Studios
Help         : Use this task to find the last used Unity Editor version of a project. An output variable containing the found version is provided. [More Information](https://unitydevops.com/docs/unity-get-project-version-task).
==============================================================================
##[warning]This task uses Node 6 execution handler, which will be deprecated soon. If you are the developer of the task - please consider the migration guideline to Node 10 handler - https://aka.ms/migrateTaskNode10. If you are the user - feel free to reach out to the owners of this task to proceed on migration.
Determining Unity editor version for project at /Users/runner/work/1/s/
Success, Unity editor version found 2018.4.36f1, alpha=false, beta=false
Finishing: Unity Get Project Version

```

**Comment by KevinBaynham** [comment]
Good day. Are they any plans to update this to Node 10? The date for deprecation is just over a month from now.

> This task uses Node 6 execution handler, which will be removed March 31st 2022. If you are the developer of the task - please consider the migration guideline to Node 10 handler - https://aka.ms/migrateTaskNode10 (check this page also if you would like to disable Node 6 deprecation warnings). If you are the user - feel free to reach out to the owners of this task to proceed on migration.

```
Starting: Unity Build
==============================================================================
Task         : Unity Build
Description  : Build a Unity project and get the exported output files.
Version      : 3.1.12
Author       : Dinomite Studios
Help         : Use this task to build your Unity project and generate the target platform's output files. [More Information](https://unitydevops.com/docs/unity-build-task)
==============================================================================
##[warning]This task uses Node 6 execution handler, which will be removed March 31st 2022. If you are the developer of the task - please consider the migration guideline to Node 10 handler - https://aka.ms/migrateTaskNode10 (check this page also if you would like to disable Node 6 deprecation warnings). If you are the user - feel free to reach out to the owners of this task to proceed on migration.
...
```

**Comment by FejZa** [comment]
Fixed in v2.9.0 of the extension package.
